src/EventStore.ClusterNode/Program.cs

```diff
@@ -231,6 +228,44 @@ internal static class Program {
 			}
 		}
 
+		private static void ConfigureHttpOptions(ListenOptions listenOptions, ClusterVNodeHostedService hostedService, bool useHttps) {
+			if (useHttps)
+				listenOptions.UseHttps(CreateServerOptionsSelectionCallback(hostedService), null);
+			else
+				listenOptions.Use(next =>
+					new ClearTextHttpMultiplexingMiddleware(next).OnConnectAsync);
+		}
+
+		private static void TryListenOnUnixSocket(ClusterVNodeHostedService hostedService, KestrelServerOptions server) {
```

Comment:
what security concerns do we potentially have here
connections from other local processes that aren't the eventstore user?
maybe its ok because its in the db directory, i wonder how it compares to the way we set permissions for checkpoints and chunks etc.

Reply:
> connections from other local processes that aren't the eventstore user?
hmmm good point, i need to verify what permissions are set by kestrel on the unix socket by default. at first glance it seems that the owner and group have read/write permissions and others have only read permissions.

Reply:
the default permission is to allow the owner and group to connect to the socket file, which seems good (in terms of security). we would thus need to document how to run the redactor app as the eventstore user.

Reply:
the only downside is that root/sudo access is required to switch to the `eventstore` user since it's not an account that can be logged into

Reply:
i think it's ok, because root/sudo access is required anyway for management purposes - for example to restart the eventstore service with `systemctl`. and as we're planning to use unix sockets for other management purposes (e.g resetting admin passwords), it seems to be fine.